strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/api/impl/ldap/LdapAuthenticator.java

```diff
@@ -0,0 +1,29 @@
+package org.carlspring.strongbox.authentication.api.impl.ldap;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.ldap.authentication.LdapAuthenticationProvider;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Component
+public class LdapAuthenticator
```

Comment:
> Also ProviderManager has a Parent->Child feature which can be very useful in our case. It can provide Authenticators Hierarchy like we already have with Spring Application Contexts. Predefined Authenticators can be placed in Parent ProviderManager, and User defined in Child ProviderManager.

In my opinion, it's not a good idea since user may order `Authenticators ` like he wants, for example:
* 1st `Authenticator `- default authenticator with predefined users [`Authenticator `supplied by us]
* 2nd `Authenticator  `- external database located in host X.Y.Z.W  [external `Authenticator `]
* 3nd `Authenticator  `- LDAP [`Authenticator `supplied by us]